e2e-tests/conf/secrets.yml

```diff
@@ -14,23 +14,3 @@ data:
   MONGODB_CLUSTER_ADMIN_PASSWORD: Y2x1c3RlckFkbWluMTIzNDU2
   MONGODB_CLUSTER_MONITOR_USER: Y2x1c3Rlck1vbml0b3I=
   MONGODB_CLUSTER_MONITOR_PASSWORD: Y2x1c3Rlck1vbml0b3IxMjM0NTY=
----
```

Comment:
But how will we test it? e.g. for PXC we have 
https://github.com/percona/percona-xtradb-cluster-operator/blob/main/e2e-tests/conf/secrets.yml
https://github.com/percona/percona-xtradb-cluster-operator/blob/main/e2e-tests/conf/secrets_without_tls.yml
Please use the same approach for psmdbo too.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1000/commits/1c9ffe22d2f7ac290ded02f3b3a784d873ad1e4c

I added `secrets_with_tls` since there were only 2 tests (init-deploy and data-sharded) that applying TLS certs before deploy works